src/model_repository_manager.h

```diff
@@ -39,42 +41,15 @@ namespace triton { namespace core {
 class InferenceServer;
 class Model;
 
+// [FIXME] should have separated load / unload functions for clarity
+enum ActionType { NO_ACTION, LOAD, UNLOAD };
```

Comment:
Do we still need NO_ACTION? When would this be needed?

Reply:
We don't need it, I just didn't modify this enum as it will be removed completely if model repository manager is going to be refactored.